src/FlowScene.cpp

```diff
@@ -351,6 +351,24 @@ connections() const
   return _connections;
 }
 
+std::vector<Node*>
+FlowScene::selectedNodes() const {
+  QList<QGraphicsItem*> graphicsItems = selectedItems();
+
+  std::vector<Node*> ret;
+  ret.reserve(graphicsItems.size());
+
+  for (QGraphicsItem* item : graphicsItems) {
+    auto ngo = qgraphicsitem_cast<NodeGraphicsObject*>(item);
```

Comment:
Did some research, found this in some qt textbook:
![image](https://cloud.githubusercontent.com/assets/7105082/23880350/b89399ac-0818-11e7-979b-7827e6299dd5.png)